calc.js

```diff
@@ -0,0 +1,73 @@
+spinnerHide();
+let x = 0;
+let y = 0;
+
+/*function fiboN(n) {
+  let a = 0, b = 1, c, i;
+  if (n == 0)
+    return a;
+  for (i = 2; i <= n; i++) {
+    c = a + b;
+    a = b;
+    b = c;
+  }
+  return b;
+} */
+
+function spinnerShow(){
+  document.getElementById('fiboResp').innerHTML = '';
+  document.getElementById('hideSpin').style.visibility = 'visible';
+  setTimeout(() => {
+    fiboCalc();
+  }, 500);
+}
+
+function fiboCalc() {  
+  x = parseInt(document.getElementById('inputN').value);
+  if (x == 42) {
+    spinnerHide();
+    document.getElementById('fiboResp').innerHTML = "42 is the meaning of life";
```

Comment:
this error should not be hard-coded. instead, you are suppode to get the error from the server.